WindowsAppSDK.Build.Cpp.props

```diff
@@ -0,0 +1,84 @@
+  <Import Project="$(MSBuildThisFileDirectory)HybridCRT.props" />
+
+  <!-- Build with APIscan-friendly compiler+linker options -->
+  <Import Project="$(MSBuildThisFileDirectory)ApiScan.Cpp.props" />
+
+  <!-- Common (all Configurations) options -->
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <!-- /std:c++17 Enable ISO C++17 Standard -->
+      <LanguageStandard>stdcpp17</LanguageStandard>
+
+      <!-- /permissive- Enable Conformance mode (i.e. Disable 'permissive' mode) -->
+      <ConformanceMode>true</ConformanceMode>
+
+      <!-- /W4 Enable max warning level -->
+      <WarningLevel>Level4</WarningLevel>
+
+      <!-- /sdk Enable additional security checks-->
+      <SDLCheck>true</SDLCheck>
+
+      <!-- /GR- Disable RTTI (causes binary bloat) -->
+      <RuntimeTypeInfo>false</RuntimeTypeInfo>
+    </ClCompile>
+    <Link>
+    </Link>
+  </ItemDefinitionGroup>
+
+  <!-- Debug-specific options -->
+  <ItemDefinitionGroup Condition="'$(Configuration)' == 'Debug'">
+    <ClCompile>
+      <!-- /d- Disable optimization -->
+      <Optimization>Disabled</Optimization>
+    </ClCompile>
+    <Link>
+    </Link>
+  </ItemDefinitionGroup>
+
+  <!-- Release-specific options -->
+  <ItemDefinitionGroup Condition="'$(Configuration)' == 'Release'">
+    <ClCompile>
+      <!-- /GS Enable Control Flow Guard -->
+      <ControlFlowGuard>Guard</ControlFlowGuard>
+
+      <!-- Compile-time optimizations:
+           /GF Enable read-only string pooling (eliminate duplicate strings)
+           /GL Whole Program Optimization
+           /Gy Enable function-level linking
+           /Gw Optimize Global Data (https://docs.microsoft.com/en-us/cpp/build/reference/gw-optimize-global-data)
+        -->
+      <StringPooling>true</StringPooling>
+      <WholeProgramOptimization>true</WholeProgramOptimization>
+      <FunctionLevelLinking>true</FunctionLevelLinking>
+      <AdditionalOptions>%(AdditionalOptions) /Gw</AdditionalOptions>
+
+      <!-- /Ox Enable most speed optimizations
+           NOTE: /Ox is short-hand for multiple options:
+                     /Ob2 Enable inline function expansion (/Ob2 = Any Suitable)
+                     /Oi Enable intrinsic functions
+                     /Ot Favor fast code
+                     /Oy Omit frame pointers
+        -->
+      <InlineFunctionExpansion>AnySuitable</InlineFunctionExpansion>
+      <IntrinsicFunctions>true</IntrinsicFunctions>
+      <FavorSizeOrSpeed>Speed</FavorSizeOrSpeed>
+      <OmitFramePointers>true</OmitFramePointers>
+    </ClCompile>
+    <Link>
+      <!-- Link-time optimizations:
+           /LCTG Enable Link Time Code Generation
+           /OPT:ICF Enable COMDAT folding
+           /OPT:REF Optimize references
+        -->
+      <LinkTimeCodeGeneration>UseLinkTimeCodeGeneration</LinkTimeCodeGeneration>
+      <EnableCOMDATFolding>true</EnableCOMDATFolding>
+      <OptimizeReferences>true</OptimizeReferences>
+    </Link>
```

Comment:
>Link [](http://example.com/codeflow?start=6&length=4)

It would probably be worth explicitly disabling incremental linking too [[https://docs.microsoft.com/en-us/cpp/build/reference/incremental-link-incrementally?view=msvc-170](https://docs.microsoft.com/en-us/cpp/build/reference/incremental-link-incrementally?view=msvc-170)](https://docs.microsoft.com/en-us/cpp/build/reference/incremental-link-incrementally?view=msvc-170).  These other flags should already be disabling it but it would be good to guarantee it.

Reply:
Good idea. It does get disabled but a few projects emit messages about disabling it because of some other setting. Explicit would prevent those messages, so yay. Thanks